### ISSUE TITLE
`Eq` instance for `Address`?

### ISSUE THREAD

**Issue by NicolasT** [evanrelf/drama#15]
Unagi's `InChan` has an `Eq` instance, so I imagine `Address` could get (a derived) one as well. This would help keeping `Address`es in some list (and removing one of them later).

**Comment by evanrelf** [comment]
I agree, that would be a useful addition. I can't think of a reason the library should forbid equating `Address`es, off the top of my head.
